src/main/java/teammates/common/util/Const.java

```diff
@@ -959,6 +959,8 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String AUTOMATED_FEEDBACKSESSION_CLOSING_MAIL_ACTION = "feedbackSessionClosingMailAction";
         public static final String AUTOMATED_FEEDBACKSESSION_OPENING_MAIL_ACTION = "feedbackSessionOpeningMailAction";
         public static final String AUTOMATED_FEEDBACKSESSION_PUBLISHED_MAIL_ACTION = "feedbackSessionPublishedMailAction";
+        public static final String AUTOMATED_FEEDBACKSESSION_UNPUBLISHED_MAIL_ACTION =
+                "feedbackSessionUnpublishedMailAction";
```

Comment:
8 space indentation is a last resort. The readability of this line can benefit from a deeper indentation.
```java
public static final String AUTOMATED_FEEDBACKSESSION_UNPUBLISHED_MAIL_ACTION =
                                                      "feedbackSessionUnpublishedMailAction";
```
Explanation: the last char of the line (i.e. `;`) is the rightmost char, as it should be.